### ISSUE TITLE
Add Swift 5.8

### ISSUE THREAD

**Issue by hobovsky** [codewars/runner#261]
- Name: Swift
- Version: 5.8
- Changelog: https://github.com/apple/swift/blob/main/CHANGELOG.md

Occasionally issues from users who use XCode locally pop up about their code not working on CW: https://www.codewars.com/kata/541c8630095125aba6000c00/discuss#64f1c002fa88aecd16185111 .

I am not sure if the language would have to be bumped all the way to 5.8, but it might be nice to have it updated at all.
Maybe it would be a good opportunity to improve Swift setup in some way, or tackle some related issues, like #244 .

---

:+1: reaction _might_ help to get this request prioritized.

**Comment by Chrono79** [comment]
I wonder if there is any other language that breaks things with every version update as much as Swift.

**Comment by hobovsky** [comment]
Update of [Assembler interpreter (part II)](https://www.codewars.com/kata/58e61f3d8ff24f774400002c/swift) is most probably blocked by this issue fixed in Swift 5.4: https://github.com/apple/swift/issues/56437

**Comment by kazk** [comment]
Swift 5.9 was added as requested by Qualified's customer. I'll run the script to update compatible kata later.
